src/ui/pages/Home/components/IdentityList/IdentityList.tsx

```diff
@@ -8,13 +8,14 @@ import {
   createIdentityRequest,
   deleteAllIdentities,
   deleteIdentity,
-  IdentityData,
   setActiveIdentity,
   setIdentityName,
   useSelectedIdentity,
 } from "@src/ui/ducks/identities";
 import { useWallet } from "@src/ui/hooks/wallet";
 
+import type { IdentityData } from "@src/types";
```

Comment:
Why are we defining `IdentityData` as interface in `@src/types`? I think it could be defined with type and we dont need to import it as a type here.